Tests/ImageFetcherTests/PerformanceTests.swift

```diff
@@ -0,0 +1,109 @@
+import XCTest
+@testable import ImageFetcher
+
+final class ImageFetcherPerformanceTests: XCTestCase {
+    enum Constants {
+        static let iterationCount = 5_000
```

Comment:
On my M1, each performance test takes about 100 seconds with this value; it that too high?